File: SIMINOV-CONNECT-TEMPLATE/src/siminov/connect/template/services/DeleteLiquor.java
package siminov.connect.template.services;

import siminov.connect.connection.design.IConnectionRequest;
import siminov.connect.connection.design.IConnectionResponse;
import siminov.connect.exception.ServiceException;
import siminov.connect.service.Service;

public class DeleteLiquor extends Service {

	private static final String SERVICE_NAME = "SIMINOV-CONNECT-LIQUORS-SERVICE";
	private static final String API_NAME = "DELETE-LIQUOR";
	
	public static final String LIQUOR_NAME = "LIQUOR_NAME";
	
	public DeleteLiquor() {
		setService(SERVICE_NAME);
		setApi(API_NAME);
	}
	
	public void onStart() {
		
	}

	public void onQueue() {
		
	}

	public void onPause() {
		
	}

	public void onResume() {
		
	}

	public void onFinish() {
		
	}

	public void onApiInvoke(IConnectionRequest connectionRequest) {
		
	}

	public void onApiFinish(IConnectionResponse connectionResponse) {
		
	}

	public void onTerminate(ServiceException serviceException) {
		
	}

	public void onRestart() {
		
	}
}

File: connect-sample/iOS/Siminov/Connect/Sample/Services/DeleteLiquor.m
//
//  DeleteLiquor.m
//  connect-sample
//
//  Created by Geetika on 29/05/15.
//  Copyright (c) 2015 Siminov. All rights reserved.
//

#import "DeleteLiquor.h"
#import "SIKIConnectionResponse.h"

static NSString * const SERVICE_NAME = @"SIMINOV-CONNECT-LIQUORS-SERVICE";
static NSString * const REQUEST_NAME = @"DELETE-LIQUOR";

static NSString * const LIQUOR_NAME = @"LIQUOR_NAME";

@implementation DeleteLiquor

- (id)init {
    
    if (self = [super init]) {
        [self setService:SERVICE_NAME];
        [self setRequest:REQUEST_NAME];
    }
    return self;
}

- (void)onStart {
    
}

- (void)onQueue {
    
}

- (void)onPause {
    
}

- (void)onResume {
    
}

- (void)onFinish {
    
}

- (void)onRequestInvoke:(id<SIKIConnectionRequest>)connectionRequest {
}

- (void)onRequestFinish:(id<SIKIConnectionResponse>)connectionResponse {
}

- (void)onTerminate:(SIKServiceException *)serviceException {
}

@end
